packages/terra-tabs/src/Tabs.jsx

```diff
@@ -176,7 +181,7 @@ class Tabs extends React.Component {
         isIconOnly = true;
       }
       clonedPanes.push(React.cloneElement(child, {
-        className: cx([{ 'is-active': isActive }, child.props.className]),
+        className: classNames(cx({ 'is-active': isActive }, theme.className), child.props.className),
```

Comment:
theme.classname should be inside the cx function

Reply:
Actually, you probably don't need to add the class here at all since it's added to the top of the component.